wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsnext/model/Container.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.workspace.server.wsnext.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Container {
+
+  private String image = null;
+  private List<EnvVar> env = new ArrayList<EnvVar>();
+  private ResourceRequirements resources = null;
+  private List<Command> commands = new ArrayList<Command>();
+  private List<Server> servers = new ArrayList<Server>();
```

Comment:
I don't remember that `Container` had a list of servers as an attribute in the original model. And I don't think that we should do that: 
- A `CheService` that has a list of `Container` and each `Container` has a list of `Servers` is something difficult to understand (what's the difference between a `CheService` and a `Server`?)
- Moreover one of workspace.next goals was to extract the list servers that originally ran in on container into multiple containers (sidecars). So it doesn't make sense to have a list of `Server` for each `Container`.

I would suggest to move higher (at `CheService` level) the attributes of the `Server`.

Reply:
I clearly see servers field in one of the model versions, in particular, https://github.com/eclipse/che/issues/9292#issuecomment-383866761.

> what's the difference between a CheService and a Server

As we discussed before usage of Service word is confusing and brings misunderstanding like this one. 

> Moreover one of workspace.next goals was to extract the list servers that originally ran in on container into multiple containers (sidecars). So it doesn't make sense to have a list of Server for each Container.

It make sense. We should, probably, discuss it with @skabashnyuk. 

> I would suggest to move higher (at CheService level) the attributes of the Server.

I don't own Workspace.Next model, so I would discuss it with Sergii first. In any case I think it makes sense to do that in a separate PR.